packages/utilities/src/Customizer.tsx

```diff
@@ -74,7 +74,8 @@ export class Customizer extends BaseComponent<ICustomizerProps, ICustomizerConte
 
   public static childContextTypes: {
     customizations: PropTypes.Requireable<{}>;
-  } = Customizer.contextTypes;
+  } =
+    Customizer.contextTypes;
```

Comment:
Hmm I've seen this on multiple of your PRs now.. I wonder if this will cause you conflicts after one gets merged